airbyte-integrations/connectors/source-looker/pyproject.toml

```diff
@@ -3,30 +3,26 @@ requires = [ "poetry-core>=1.0.0",]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
-version = "0.2.12"
+version = "1.0.0"
 name = "source-looker"
-description = "Source implementation for Looker."
+description = "Source implementation for looker."
 authors = [ "Airbyte <contact@airbyte.io>",]
 license = "MIT"
 readme = "README.md"
 documentation = "https://docs.airbyte.com/integrations/sources/looker"
 homepage = "https://airbyte.com"
 repository = "https://github.com/airbytehq/airbyte"
-[[tool.poetry.packages]]
-include = "source_looker"
+packages = [ { include = "source_looker" }, { include = "main.py" } ]
 
 [tool.poetry.dependencies]
 python = "^3.9,<3.12"
-airbyte-cdk = "0.80.0"
-prance = "==0.21.8.0"
-openapi-spec-validator = "==0.3.3"
-types-requests = "==2.28.11.7"
+airbyte-cdk = "^3.8.2"
```

Comment:
Oh, good! Makes sense to jump to a recent CDK. While this is in review, you might as well bump this to CDK 4.0.1, but if you choose to do this, don't forget to `poetry lock` as well.